eap74-rhel8-byos-multivm/src/main/scripts/jbosseap-setup-master.sh

```diff
@@ -0,0 +1,278 @@
+#!/bin/bash
+
```

Comment:
Why no

```
#!/usr/bin/env bash
set -Eeuo pipefail
```

Reply:
`-e` will exit the script at once, we will lose logs in commands like:
```
subscription-manager register --username $RHSM_USER --password $RHSM_PASSWORD --force | log; flag=${PIPESTATUS[0]}
```
Considering commands like above, if we do not use  `-e`, we should probably not use `-E` and `-o pipefail` either
`-u` will cause us trouble, because command 
```
source ~/.bash_profile
```
will call `/etc/bashrc` in RHEL8, and lots of variables are not defined in it, for example in line 12:
```
1  # /etc/bashrc
2
3  # System wide functions and aliases
4  # Environment stuff goes in /etc/profile
5
6  # It's NOT a good idea to change this file unless you know what you
7  # are doing. It's much better to create a custom.sh shell script in
8  # /etc/profile.d/ to make custom changes to your environment, as this
9  # will prevent the need for merging in future updates.
10
11  # Prevent doublesourcing
12  if [ -z "$BASHRCSOURCED" ]; then
13    BASHRCSOURCED="Y"
14
15    # are we an interactive shell?
16    if [ "$PS1" ]; then
17      if [ -z "$PROMPT_COMMAND" ]; then
18        case $TERM in
19        xterm*|vte*)
20          if [ -e /etc/sysconfig/bash-prompt-xterm ]; then
21              PROMPT_COMMAND=/etc/sysconfig/bash-prompt-xterm
22          elif [ "${VTE_VERSION:-0}" -ge 3405 ]; then
23              PROMPT_COMMAND="__vte_prompt_command"
24          else
25              P
...
```

Reply:
I polished the code a bit more with more logging: https://github.com/Azure/rhel-jboss-templates/pull/96/commits/e90aba13d4519c2544e208ea9153dda017d4d5bf